python/pyspark/profiler.py

```diff
@@ -459,6 +459,12 @@ def stats(self) -> CodeMapDict:
     def _show_results(
         code_map: CodeMapDict, stream: Optional[Any] = None, precision: int = 1
     ) -> None:
+        if has_memory_profiler:
```

Comment:
hmm, should this be `if not has_memory_profiler` ?

Reply:
oh! yeah ... wonder why my manual test was fine. Let me test it out again

Reply:
oops, it was my bad.